apps/web/src/app/(docs)/docs/sandbox/list/page.mdx

```diff
@@ -95,20 +96,21 @@ from e2b_code_interpreter import Sandbox
 # Create sandbox with metadata.
 sandbox = Sandbox(
     metadata={
-        "env": "dev",  # $HighlightLine
-        "app": "my-app",  # $HighlightLine
-        "user_id": "123",  # $HighlightLine
+        "env": "dev", # $HighlightLine
+        "app": "my-app", # $HighlightLine
+        "user_id": "123", # $HighlightLine
     },
 )
 
 # List running sandboxes that has `userId` key with value `123` and `env` key with value `dev`.
 running_sandboxes = Sandbox.list(
```

Comment:
incorrect syntax

Reply:
are you sure? this is for creating the sandbox, not querying...

Reply:
I meant lines 107--113